MPChartLib/src/com/github/mikephil/charting/interfaces/datasets/IDataSet.java

```diff
@@ -392,4 +392,16 @@
      * @return
      */
     boolean isVisible();
+
```

Comment:
As I think about it, perhaps it would be better to call this object ImageCache rather than UserData, since that would prevent confusion regarding when or where the object is intended to be used.